rir/src/compiler/translations/rir_2_pir/stack_machine.h

```diff
@@ -30,7 +30,10 @@ class StackMachine {
 
     typedef std::pair<BB*, Value*> ReturnSite;
     typedef std::function<void(ReturnSite)> ReturnMaybe;
-    void runCurrentBC(const Rir2Pir& cmp, Builder&);
+
+    typedef std::function<void()> Maybe;
```

Comment:
What's our style guide / consensus on `typedef` vs `using`?

Because of old habits, I tend to use `typedef` more often, but `using` lets you use templates. Also, something like

```
using foobar = void (*)(double);
```

is a bit easier to read than

```
typedef void (*foobar)(double);
```